.golangci.yml

```diff
@@ -154,6 +155,12 @@ linters-settings:
     locale: US
     ignore-words:
       - cancelled
+  perfsprint:
```

Comment:
Can this be configured to not check test files?

Reply:
Performance is not critical for tests and it provides an alternate way to test in a functionally equivalent way.

Reply:
@mmorel-35 I think you can simply add `perfsprint` to https://github.com/open-telemetry/opentelemetry-go/blob/0d48e02ad81af1feb5d6e6ceeb70e0a05c0326d6/.golangci.yml#L65-L68